jetty-server/src/main/java/org/eclipse/jetty/server/HttpChannelOverHttp.java

```diff
@@ -74,6 +75,17 @@ public HttpChannelOverHttp(HttpConnection httpConnection, Connector connector, H
         _httpConnection = httpConnection;
     }
 
+    @Override
+    protected void execute(Runnable task)
+    {
+        if (_httpConnection.isUseVirtualThreadToInvokeRootHandler())
+        {
+            if (VirtualThreads.startVirtualThread(task))
+                return;
+        }
+        super.execute(task);
+    }
```

Comment:
Is this using virtual threads too much?   It is certainly using them more than the claim in the current name of "InvokeRootHandler".